akka-docs-dev/rst/scala/code/docs/http/scaladsl/server/directives/PathDirectivesExamplesSpec.scala

```diff
@@ -16,7 +16,7 @@ class PathDirectivesExamplesSpec extends RoutingSpec {
 
   //# path-dsl
   // matches /foo/
-  path("foo" /)
+  path("foo"./)
```

Comment:
it's no problem with postfix when used within `( )` as here and I thought it was a convention to use it like that in the routing dsl.

I'm fine with `"foo"./`